sdk/sql/mgmt/src/main/java/com/azure/management/sql/AdministratorType.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
what's this file from? new generaton?

Reply:
Yes, all header with `Code generated ...` is from generator.